### PR TITLE
Update format to include more types

### PR DESCRIPTION
**Pull request by karikkato**
added bool as well as ts2 to have a string formatted datetime, and a text field allowing for a list of predefined strings, useful for things such as aggregation testing

also updated to allow for a set length, as before using the same number for both min and max would cause an exception in the range function

### CODE REVIEW

File: es_test_data.py [hunk 1]
```diff
@@ -70,16 +71,26 @@ def get_data_for_format(format):
...
     field_type = split_f[1]
 
+    if field_type == "bool":
+        return_val = random.choice([True, False])
+
     if field_type == "str":
         min = 3 if len(split_f) < 3 else int(split_f[2])
         max = min + 7 if len(split_f) < 4 else int(split_f[3])
-        length = random.randrange(min, max)
+        if min == max:
```
Comment (oliver006) [inline es_test_data.py]: Can you move this into a separate function? Looks like it's used in a bunch of places now.

File: es_test_data.py [hunk 2]
```diff
@@ -88,10 +99,22 @@ def get_data_for_format(format):
         max = now + 30 * per_day if len(split_f) < 4 else int(split_f[3])
         return_val = int(random.randrange(min, max) * 1000)
 
+
+    elif field_type == "ts2":
```
Comment (oliver006) [inline es_test_data.py]: Want to call this `tstxt` instead? `ts2` works as well but could be a bit more descriptive.

File: es_test_data.py [hunk 3]
```diff
@@ -88,10 +99,22 @@ def get_data_for_format(format):
...
+        now = int(time.time())
+        per_day = 24 * 60 * 60
+        min = now - 30 * per_day if len(split_f) < 3 else int(split_f[2])
+        max = now + 30 * per_day if len(split_f) < 4 else int(split_f[3])
```
Comment (oliver006) [inline es_test_data.py:103]: You could consolidate this with the `ts` generation somewhat like this:

```
if field_type in [ts, ts2]:
    min = ...
    max = ...
    ts = int(randrange(...))
    return_val = ts if field_type == 'ts' else return_val = fromtimestamp(ts) 
```

**Comment by oliver006** [comment]
Thanks for the pull request, I like the new field types!
Two small comments, let me know what you think and we should be good to go.

**Comment by karikkato** [comment]
Glad you enjoyed it, it was my first foray into python, and I really needed some random data :)

I'll try to see if I have some time later today to make those changes

**Comment by oliver006** [comment]
Awesome, thanks, looking forward to it!

**Comment by karikkato** [comment]
Take a look at it now, I might try to add array support later since I have a use case for it.

**Comment by oliver006** [comment]
I like it, nice work!
Can you resolve the merge conflicts and update your branch and we should be good to get this merged!

**Comment by karikkato** [comment]
Not sure if you're waiting on a response from me, but you can merge now

**Comment by oliver006** [comment]
Merged, thanks for the reminder!

**Pull request merged**